src/07_slices.py

```diff
@@ -12,26 +12,27 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
 
 # Output the second-to-last element: 9
-print()
+print(a[len(a) - 2: len(a) - 1])
```

Comment:
This works. You could also take advantage of negative indexing to work off of the end of the list.